nexus/src/external_api/device_auth.rs

```diff
@@ -125,7 +125,6 @@ pub struct DeviceAuthVerify {
 }]
```

Comment:
(see line above) kind of think this should be `unpublished = true` instead of `tags = ["hidden"]`, what do you think @plotnick?

Reply:
Yeah, @ahl suggested that, too. It's fine with me, since we don't use most of these endpoints from our client code at all. But we do use `/device/confirm` from console code, so we should choose a tag or use `hidden` for that one.